### ISSUE TITLE
[Crash]: Upgraded to 2.11 and client crashes 

### ISSUE THREAD

**Issue by mfczureal** [AllTheMods/ATM-10#1547]
### Possible Fixes

Yes

### Modpack Version

2.11

### What happened?

Server is running on Linux and is not crashing, it's just the client-side crashing and forcing a restart of the MC client.  I've got the client set for 8GB RAM on start and I've reinstalled 2.11 twice and rebooted at least once, but the client keeps crashing on occasion.  I never had this problem on any of the earlier versions, so this is new.  

### Crash Log

https://gist.github.com/mfczureal/a0464584a48d6723e79264237fbdb637

**Comment by alekthefirst** [comment]
If you can, add this mod
https://www.curseforge.com/minecraft/mc-mods/betterrandomsourceconcurrencycrash
and get a crash log, the normal ones like the one you gave doesn't really have much useful information

**Comment by mfczureal** [comment]
[crash-2024-12-26_11.07.36-client.txt](https://github.com/user-attachments/files/18253958/crash-2024-12-26_11.07.36-client.txt)


Installed the mod and crashed.  I've attached the crash-report, hopefully this has the right information for you.  Thanks!

**Comment by mfczureal** [comment]
Crashed again just a few minutes later, figured I'd drop the other crash report in here too. 
[crash-2024-12-26_11.12.33-client.txt](https://github.com/user-attachments/files/18253983/crash-2024-12-26_11.12.33-client.txt)

**Comment by alekthefirst** [comment]
Seems to be mekanism covers
in `config\mekanismcovers.json` set `disableAdvancedCoverRendering` to true

**Comment by mfczureal** [comment]
> Seems to be mekanism covers in `config\mekanismcovers.json` set `disableAdvancedCoverRendering` to true

is that on the client side or the server side?

**Comment by alekthefirst** [comment]
client side

**Comment by mfczureal** [comment]
> client side

I opened up the config file and it was already set to true.

**Comment by alekthefirst** [comment]
Well that's weird and concerning, you are the only one with this issue that i know of at this moment
do you use anything from mekanism covers?

**Comment by mfczureal** [comment]
I do have some mek covers in my main base on some mek cables.  I run the server too, so if there's a command I can run to remove them all from the world, I'm okay with going down that route too.

**Comment by mfczureal** [comment]
I removed the mod from the server and am restarting the server, hopefully that will resolve at least my immediate problems.

**Comment by Mawarii** [comment]
I am having the same issue. I also have some covers in my base.

**Comment by mfczureal** [comment]
> I am having the same issue. I also have some covers in my base. How can I help fix this issue?

I removed mekanismcovers mod from my server, restarted server and that removed them from anyone using them and I haven't had a crash since.

**Comment by hamodcraft22** [comment]
i had the same issue with crashing but for me a chunk which was loaded in the mining diemnetion which had a builder from RF tools.

unloading the chunk seems to have stopped the crashing (the crash report is essentially the same as the one uploaded here)

![image](https://github.com/user-attachments/assets/11545bcf-87aa-4d68-8823-fa5bcc1b981d)

**Comment by hamodcraft22** [comment]
yep same situation as well, removed the mod "mekanismcovers" from the server and it seemed to resolve the issue
